cura/PrinterOutput/PrintJobOutputModel.py

```diff
@@ -147,4 +151,14 @@ def updateState(self, new_state):
 
     @pyqtSlot(str)
     def setState(self, state):
-        self._output_controller.setJobState(self, state)
\ No newline at end of file
+        self._output_controller.setJobState(self, state)
+
+    @pyqtProperty("QVariantList", notify=configurationChangesChanged)
+    def configurationChanges(self) -> List[ConfigurationChangeModel]:
+        return self._configuration_changes
+
+    def updateConfigurationChanges(self, changes: List[ConfigurationChangeModel]) -> None:
+        if len(self._configuration_changes) == 0 and len(changes) == 0:
+            return
+        self._configuration_changes = changes
```

Comment:
Missing check to see if something changed. The current implementation will ensure that for every printjob with config changes, you will get an emit every 2 seconds (forcing a redraw in qml)